dataclass_csv/dataclass_reader.py

```diff
@@ -155,6 +167,11 @@ def _process_row(self, row):
                     values.append(transformed_value)
                     continue
 
+            if field.type is bool:
```

Comment:
Hi @schobster , you thing that I forgot to mention.

The way that the other types are implemented are:
If the value is not present it will try to get the default value of the dataclass field. If the field in the dataclass doesn't have a default (required fields), then an exception is raised saying the error and the line of the csv file that the error occurred. So we would need to do something similar to this:
```python
if field.type is bool:
    try:
        transformed_value = (
            value
            if isinstance(value, bool)
            else strtobool(value.strip()) == True
        )
    except ValueError as ex:
        raise CsvValueError(
            ex, line_number=self.reader.line_num
        ) from None
    else:
        values.append(transformed_value)
        continue
```
When I set the `transformed_value` I added that if statement, if the value is instance of bool it means that it has got the dataclass field's default value, otherwise it is a string and needs to be converted to bool. Note that I also used the strip function, just to make sure we clear the empty spaces in both sides of the string.

Then in the end I do  a `strtobool(value.strip()) == True`, that's because `strtobool` returns 0 or 1 and doing so will return True or False.

Reply:
Ah I see @dfurtado , thank you for pointing this out and for making things very clear - I'll update this updated to follow the convention used by the other types.